### DISCUSSION TITLE
Multiple apps deploy on vercel

### DISCUSSION THREAD

**Discussion by vitoorgomes** [vercel/turborepo#2629] [Help] [answered]
if I have a structure like this:

```
├── root
├── apps
│   ├── app1
│   │   ├── ...
│   │   ├── .next
│   │   ├── .turbo
│   │   ├── package.json
│   ├── app2
│   │   ├── ...
│   │   ├── .next
│   │   ├── .turbo
│   │   ├── package.json
├── node_modules
├── package.json
├── yarn.lock
└── turbo.json
```

is it possible to have one repository holding all of that, integrate with the vercel dashboard and deploy `app1` and `app2` on different domains? Also, if I commit changes to `app1`, is possible to control when the deploy need to happen only `app1`?

Is there any tutorial/docs or something on this topic? I can't seem to find anything related, only deploys choosing one of the folders inside `apps` at the vercel dashboard

**Comment by EarlGeorge** [comment, marked as answer]
https://vercel.com/docs/concepts/monorepos

**Reply by vitoorgomes** [reply]
thanks Earl, thats on me for not carefully reading the docs, I was able to clone each project inside the monorepo but was having some trouble while trying to ignore the build step. Had some time to rest my mind and take it step by step and got it working, thanks :)
